examples/result/src/main.sw

```diff
@@ -0,0 +1,21 @@
+script;
+
+enum MyContractError {
+    DivisionByZero: (),
+}
+
+fn divide(numerator: u64, denominator: u64) -> Result<u64, MyContractError> {
+    if (denominator == 0) {
+        return Result::Err(MyContractError::MyErrorMessage);
```

Comment:
Wait.. how is CI passing even though the error variant name is now different? Shouldn't this be `DivisionByZero`?

Reply:
that's weird... but I fixed it!

Reply:
Oh I just realized we have a `Forc.toml` file for the `examples` workspace which needs to be updated. I'll push to your branch.